src/pages/projectSettings/tabs/PluginsTab/PluginsTab.tsx

```diff
@@ -52,10 +51,7 @@ const validate = (formData, errors) => {
   const searchProject = formData?.buildBaronSettings?.ticketSearchProjects;
 
   // if a search project is defined, a create project must be defined, and vice versa
-  const searchProjectDefined =
-    searchProject?.length !== 0 &&
-    searchProject?.[0]?.searchProject !== undefined &&
-    searchProject?.[0]?.searchProject.trim() !== "";
+  const searchProjectDefined = searchProject?.length !== 0;
```

Comment:
can searchProject be undefined? 
`(undefined !== 0) === true`

Reply:
No, it will always [default to `[]`](https://github.com/evergreen-ci/spruce/blob/d3b68d1821f8500ae81d6663578368f2beef143b/src/pages/projectSettings/tabs/PluginsTab/transformers.ts#L28-L31)

I also added typing to the validate function!